src/v2/ion/ui-schema/ion-object-handler.js

```diff
@@ -147,6 +147,9 @@ const createUiSchemaForObject = (ionFormField, remediationForm, transformedResp,
   }
 
   return uiSchema;
-};
+}
 
-export default createUiSchemaForObject;
+export {
+  createUiSchemaForObject as default,
+  createOVOptions
+};
```

Comment:
Exporting an object will actually import the whole object into the bundle as compared to individual exports. In order to support tree-shaking we would need something like
```
export function createOVOptions() {}
.
.
.
export default createUiSchemaForObject;
```

Reply:
`export default` make sense..